app/pages/blackout/what_is_mhacks.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import { SectionHeader, SectionBody } from './section_components.jsx';
+
+import { Container } from '../../components';
+import { devices } from '../../styles';
+
+const Orb1 = require('../../../static/blackout/Orb 1.png');
+
+const OrbImage1 = styled.img`
+    height: 500px;
+    width: auto;
+
+    ${devices.tablet`
+        position: relative;
+        left: -20%;
+    `}
+
+    ${devices.giant`
```

Comment:
Since these are same styles as desktop you don't have to redeclare them for giant